### PR TITLE
Add rules for merging pull requests to the docs

### PR DESCRIPTION
**Pull request by Arnei**
What a committer is and is not allowed when merging a pull request is not clearly documented. The best way to inform myself about the rules I could currently find is sifting through the proposal logs, which is time consuming, confusing and makes it easy to miss relevant proposals. Therefore, I would like to add the rules in a place in the docs where they are easier to find.

### Your pull request should…

* [ ] have a concise title
* [ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [ ] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [ ] have a clean commit history
* [ ] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: docs/guides/developer/docs/development-process.md [hunk 1]
```diff
@@ -86,6 +86,21 @@ modules/admin-ui/src/main/java | In case the interface of the Admin UI facade ch
 While a committer may accept a patch even if it does not meet these expectations, it is encouraged that anyone filing
 a pull request ensures that they meet these expectations.
 
+#### Merging Pull Requests
+
+After a pull request has received at least one approving review and passes the automated tests, it is ready for merging.
+Only a committer can perform a merge, so if a reviewed pull request has not yet received attention from a committer
+feel free to contact one.
```
Comment (gregorydlogan) [inline docs/guides/developer/docs/development-process.md:93]: Who's `me`?

File: docs/guides/developer/docs/development-process.md [hunk 2]
```diff
@@ -86,6 +86,21 @@ modules/admin-ui/src/main/java | In case the interface of the Admin UI facade ch
...
+
+There are a couple of rules that committers must follow when merging pull requests. These are:
+* You cannot merge your own pull requests.
```
Comment (gregorydlogan) [inline docs/guides/developer/docs/development-process.md]: This rule has been dropped. It's discouraged, but as long as it's been talked about at the technical meeting then it's free game to merge.

File: docs/guides/developer/docs/development-process.md [hunk 3]
```diff
@@ -86,6 +86,21 @@ modules/admin-ui/src/main/java | In case the interface of the Admin UI facade ch
...
+* A pull request requires at least one approving review before merging.
+    * More reviews are always welcome.
+* You cannot review your own pull requests.
```
Comment (gregorydlogan) [inline docs/guides/developer/docs/development-process.md]: Duplicate of the line above. Same issue :)

**Review by gregorydlogan — CHANGES_REQUESTED** [review]

File: docs/guides/developer/docs/development-process.md (hunk 3 above)
Comment (Arnei) [reply docs/guides/developer/docs/development-process.md]: Oh really? So a committer can create, review and merge their own PR, as long as the people in the technical meeting gave their ok?

File: docs/guides/developer/docs/development-process.md (hunk 1 above)
Comment (Arnei) [reply docs/guides/developer/docs/development-process.md:93]: Are you suggesting to put a link to the list of current committers there?

Comment (gregorydlogan) [reply docs/guides/developer/docs/development-process.md:93]: No, I'm asking who that sentence is referring to - me or Lars I'm guessing, but someone not as familiar with the community wouldn't have a clue.

Edit: either this was re based or I misread this. Nevermind :)

File: docs/guides/developer/docs/development-process.md (hunk 3 above)
Comment (gregorydlogan) [reply docs/guides/developer/docs/development-process.md]: Technically correct, the best kind of correct. I would strongly discourage it, but it's allowed.

**Review by gregorydlogan — APPROVED** [review]

**Pull request merged**